integration_tests/mockApis/nomisMigrationApi.ts

```diff
@@ -447,7 +447,7 @@ const stubHealth = (): SuperAgentRequest =>
               messagesInFlight: '0',
               dlqStatus: 'UP',
               dlqName: 'dps-syscon-dev-visitsmigration_dlq',
-              messagesOnDlq: '153',
+              messagesOnDlq: `${failures}`,
```

Comment:
surprised that intellij allowed this - shouldn't it just be
```suggestion
              messagesOnDlq: failures,
```